doc/ref_cert/RC1/chapters/chapter04.md

```diff
@@ -233,7 +233,7 @@ ansible-playbook functest-src/ansible/site.cntt.yml
 ```
 
 <a name="4.3.2"></a>
-### 4.3.2 Run CNTT NFVI Testing
+### 4.3.2 Run CNTT Cloud Infrastructure Testing
```

Comment:
```suggestion
### 4.3.2 Run Cloud Infrastructure Testing
```

Reply:
I still think this causes confusion between RC1 and RC2.  RC2 is more like "Run Kubernetes conformance suite."  So it would seem like this should reference either NFVI (to me implies OpenStack testing), or referring specific to OpenStack, which is the the target of Rx1, while Kubernetes is the target of Rx2.

Reply:
OpenStack testing works for me. I made a correction.